src/main/kotlin/cc/hyperium/game/huds/modules/TextModule.kt

```diff
@@ -0,0 +1,28 @@
+package cc.hyperium.game.huds.modules
+
+import cc.hyperium.game.huds.ColorScheme
+import me.kbrewster.blazeapi.client.fontRenderer
+
+class TextModule(name: String, description: String, width: Int, length: Int = 0) :
+        StandardModule(name, description, length, width) {
+
+    var shadow: Boolean = false
+
+    override fun render(scheme: ColorScheme, x: Int, y: Int) {
+        if (shadow) {
```

Comment:
Isn't there a method that accepts shadow as a boolean? If so, lets just pass that through.